app/controllers/your-psp/yourpsp-tasks.service.test.js

```diff
@@ -0,0 +1,69 @@
+const { expect } = require('chai')
+const { getCurrentCredential } = require('../../utils/credentials')
+const { getNewStripeTaskList, stripeTaskListisComplete } = require('./yourpsp-tasks.service')
+const gatewayAccountFixtures = require('../../../test/fixtures/gateway-account.fixtures')
+
+describe('Progress of Stripe Psp tasklist page', () => {
+  it('correctly calculates all conditions being met for Stripe', () => {
+    const account = gatewayAccountFixtures.validGatewayAccount({
+      gateway_account_credentials: [
+        { state: 'ACTIVE', payment_provider: 'stripe', id: 100 }
+      ]
+    })
+
+    account.connectorGatewayAccountStripeProgress = {
+      bankAccount: true,
+      director: true,
+      vatNumber: true,
+      companyNumber: true,
+      responsiblePerson: true,
+      organisationDetails: true,
+      governmentEntityDocument: true
+    }
+    const activeCredential = getCurrentCredential(account)
+    const taskList = getNewStripeTaskList(activeCredential, account)
```

Comment:
Can some assertions be added for the task list which can check if `enabled & complete` values are correctly set for each task?